### PR TITLE
added onClick event 

### PR DESCRIPTION
**Pull request by JClutch**
Added a content script to run an onClick listener for both window and document events (don't know yet which one we're going to want to register specifically)

### CODE REVIEW

**Comment by JClutch** [comment]
https://developer.chrome.com/extensions/content_scripts

Some reference for content scripts

File: chrome/manifest.json [hunk 1]
```diff
@@ -9,6 +9,13 @@
   "page_action": {
     "default_popup": "views/popup.html"
   },
-  "permissions": ["declarativeContent"],
+  "permissions": ["declarativeContent", "activeTab"],
+  "content_scripts": [
+    {
+      "matches": ["http://*/*","https://*/*"],
```
Comment (JClutch) [inline chrome/manifest.json:15]: This is to have the content script run on as many sites as possible. If we ever decide to extend to files we have to add that type here

File: chrome/js/onClick.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+window.addEventListener("click", function(event) {
+  console.log('were in the clikc WINDOW', event)
```
Comment (pengcheng95) [inline chrome/js/onClick.js:3]: nit: spelling click

File: chrome/js/onClick.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+});
+
+document.addEventListener("click", function(event) {
```
Comment (pengcheng95) [inline chrome/js/onClick.js:6]: Would we want the event listener in both window and document?

Comment (JClutch) [reply chrome/js/onClick.js:6]: Unsure, that's why I left both there (see top comment)

File: chrome/js/onClick.js (hunk 2 above)
Comment (JClutch) [reply chrome/js/onClick.js:3]: Nit: I hate your nit

**Pull request merged**